contracts/vaults/PlasmaVault.sol

```diff
@@ -652,7 +653,16 @@ contract PlasmaVault is
 
         _realizeManagementFee();
 
-        return super.mint(shares_, receiver_);
+        (address feeRecipient, uint256 feeShares) = PlasmaVaultFeesLib.prepareForRealizeDepositFee(shares_);
+
+        uint256 depositAssets = super.mint(shares_ + feeShares, receiver_);
```

Comment:
user "uint256 depositAssets" in returns and then you don't have to declare this variable again